src/store-modules/types.d.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * The search result object
+ */
+export interface MediaResult {
+  result_count: number
+  page_count: number
+  page_size: number
+  results: AudioDetail[] | ImageDetail[]
+}
```

Comment:
If we can make this a generic type I think it will be more useful. For example if someone calles `getAudio` it should return a `MediaResult<AudioDetail>` without them having to do a typecheck on the `results` property right?